### DISCUSSION TITLE
[TK] Proposal of tuning changes to make the raid more fluid and enjoyable

### DISCUSSION THREAD

**Discussion by shaszun** [chromiecraft/chromiecraft#6854] [Suggestions]
There were some nice ideas in the CC discord channels regarding TK tuning. I will try to combine them in this topic.

Trash - in my opinion overall the dmg is good, it's not tank and spank like SSC. Raid need to coordinate and utilize skills that were gathering dust in people's spellbooks and barely used in PvE on this server. Including even in heroic dungeons. Trash's HP is overall good, but some of it can be tweeked.

- Hawks and Robots HP to be nerfed from around 2.3M to around 1.5-1.6M or something similar. If you ask why? You fight these separated from any other type of trash and they barely have any mechanics. They are just big sponges of HP that slows down the raid and no one likes to spend  75% of their raiding in clearing only trash. With time people will get better and faster, but these 2 will be just getting more annoying to deal with, because of the wasted time.
- Solarian - Currently it is very high DPS and healing check. I can see a lot of the guilds having problems on this boss with their secondary raids. Not gonna comment about pugs... In my opinion Solarian's Arcane Missiles dmg needs a bit of nerfing, around 10-20%. You need really good healers with fast reaction in it's current state and in case of Silence more or less you can lose multiple people very fast. Before people start saying that you can avoid the Silence. I'm aware, but people make mistakes and in this fight feels like it's on Mag cube mechanic level. Adds HP also can get a bit of cut similar to the AM 10-20% to give fighting chance to the more casual guilds/raids.
- KT - Solaris proved it's killable, so there aren't any damage issues that will be problematic from what I know. Currently there is a bug with guards HP (https://github.com/chromiecraft/chromiecraft/issues/6851). In its current state it looks decent enough to be close with transition timers. Of course this will become way easier with time, but if this gets fixed. It will require retuning of their HP for P1 and P3. If they get their current HP doubled in P3, I think it's impossible to transition in P4 with all guards killed. My best guess would be 2.5 guards killed and P4 will start. I don't know the exact numbers but I think they had 800k each right now, doubling that would be 1.6M, times 4 equals 6.4M HP in 3 minutes with a lot of DPS lose due to running around between them. Sounds like a headache.

New additions to the topic:
-Al'ar spawns doing small hit damage - result is, you can ignore the spawns and just focus the boss the whole fight. Mechanically you never were supposed to do that unless in the zerg phase (last 10-20%)
-KT phoenix's egg hp - it had way less hp on the PTR (50k) compared to live (500k). This might need to be retuned again after MC related fixes are applied. Currently sounds like it's too much.

Suggestion:
To get the expected Live tuning few days prior of release on the PTR. This way the community will be able to give more informative opinion and fine tuning to be done even before release.

**ETA:  I misinterpreted the report of the guards, but I will leave the initial text as it is.**
**ETA2:  New additions and Suggestion were added..**

Add your opinions bellow, if you support what I wrote click the like and subscribe buttons.

**Comment by balleny** [comment]
> Hawks and Robots HP to be nerfed from around 2.3M to around 1.5-1.6M or something similar. If you ask why? You fight these separated from any other type of trash and they barely have any mechanics. They are just big sponges of HP that slows down the raid and no one likes to spend 75% of their raiding in clearing only trash. With time people will get better and faster, but these 2 will be just getting more annoying to deal with, because of the wasted time.

i think it would be fair to reduce the HP a bit, but at the same time, the first occurence of the abilities each npc has should be reduced the same way. otherwise, you will kill sentinals before they even try to cast their explosion and hence could literally remove the mobs at all (if they die before using any ability). But, thats a change to the mechanics and unlikely to happen (same with pre-nerf versions not being implemented). Hence, I would steer away from adjusting at least the robots.

> Solarian - Currently it is very high DPS and healing check. I can see a lot of the guilds having problems on this boss with their secondary raids. Not gonna comment about pugs... In my opinion Solarian's Arcane Missiles dmg needs a bit of nerfing, around 10-20%. You need really good healers with fast reaction in it's current state and in case of Silence more or less you can lose multiple people very fast. Before people start saying that you can avoid the Silence. I'm aware, but people make mistakes and in this fight feels like it's on Mag cube mechanic level. Adds HP also can get a bit of cut similar to the AM 10-20% to give fighting chance to the more casual guilds/raids.

this fight and voidreaver, with everyone stacked below him, are the first fights we have in TBC that are not borderline boring as a healer but you actually get to enjoy putting out heal all fight long (voidreaver) or have to react quick/use targetoftarget (solarian). Further, I am not aware of any guild that attempted solarian but didnt manage to kill her yet - sample size may be low tho - but thats likely true for your ASSUMPTIONS of guilds struggling on it, too.
the only need for adjustment that I see is the exploding debuff that can crit and results in oneshots even through PW:S etc. see also https://github.com/chromiecraft/chromiecraft/issues/6850 (mages / priests / warlocks wont survive 14k crits from this explosion yet without avoiding it in total by usage of iceblock / dispersion)

> KT - Solaris proved it's killable, so there aren't any damage issues that will be problematic from what I know. Currently there is a bug with guards HP (https://github.com/chromiecraft/chromiecraft/issues/6851). In its current state it looks decent enough to be close with transition timers. Of course this will become way easier with time, but if this gets fixed. It will require retuning of their HP for P1 and P3. If they get their current HP doubled in P3, I think it's impossible to transition in P4 with all guards killed. My best guess would be 2.5 guards killed and P4 will start. I don't know the exact numbers but I think they had 800k each right now, doubling that would be 1.6M, times 4 equals 6.4M HP in 3 minutes with a lot of DPS lose due to running around between them. Sounds like a headache.

adds have 800k / 800k life in p1 and 800k / 1600k in p3. that ticket just describes that they are supposed to have 800k / 800k in p3, too.

**Reply by shaszun** [reply]
1. You are somewhat right about that, for now it's fine, but it will become tedious with time.
2. If it's a crit no one except a tank will survive it, you can also add the RNG elements of the explosion and AM or the other AOE spell exactly after it resulting of 100% dead person. By **ASSUMPTIONS** I said secondary raids, ofc most of the guilds will get a group that can do 40k+ dps overall for the fight and manage it. Right now the guilds more or less went with 1 raid having most of their top members inside TK and it was somewhat of a challenge. Most guilds run multiple raids and these people will spread with their respective schedule so quality will drop in some degree. I won't say guild names, but some are currently in a state that they said they won't enter TK, because they have issues with Vashj. And NO, I'm not in a such guild. Other guilds by looking their legacy stats also will have big issues regarding either dps or healing department.
3. Ok, misinterpreted it.

**Reply by Krateozshael** [reply]
Rum Rage went yesterday and we are able to first try every Boss (except KT ofc). I also think that the tuning of Solarian mainly is too much of a dps and heal check for less optimized guild.
The random oneshots of this fight caused us to sweat really hard at the end with barely making the dps check on the adds. Therefore this boss definitivly needs some changes, either in fixing "bugs" or tuning down add-hp by like 10-15%

For Trash:
At the end its just too much of a time waste in the long run. It will be just as tedious as doing trash in SSC.
Im fine with smashing my head against them for like 1 month to get rid of any bugs, but then i highly suggest nerfing hp drastically (30-40% at least) 
At the end you want to be appealing for newcomers to CC, and having to fight 1 1/2 hours of trash is certainly not fun

I dont need to elaborate on KT, this boss is just a mess as it stands rn. Nerf HP of adds, get the walking speed of taladred done and nerf bomb damage (15k on tanks are a big yikers)

-Shael

**Comment by Zelfie** [comment]
Trash is boring. Takes way too long.
Nerf the absolute shit out of it.
"It'll get faster with gear/efficiency" - maybe but nah, i'd rather just play something else TBH. Feels disheartening to spend so long burning through sponge-trash to get to 1shot bosses.

**Reply by balleny** [reply]
You know that half the fights in TK are tactically less demanding than trash?

**Reply by Zelfie** [reply]
Yes.

**Comment by shaszun** [comment]
The topic was updated with 2 additional tunning problems and a suggestion.

**Comment by Jbubby** [comment]
"you know what would make this raid great? If the metric fuck ton of trash mobs took a century to clear" 

No one has actually ever said this btw, PLS detune some of these mfs
